test/e2e/app-dir/ecmascript-features/app/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+import { abc } from './export-as'
+import json from './file.json' with { type: 'json' }
+// Test both server and client compilation of ECMAScript features.
+
+class ClassWithPrivate {
+  #privateField
+  #privateFieldWithInitializer = 11
+
+  #privateMethod() {
+    this.#privateField = 10
```

Comment:
```suggestion
    this.#privateField = 10
    #privateField in this;
```

Reply:
Added! https://github.com/vercel/next.js/pull/65613/files#diff-b8678af04064ee7c1e38b7412725f6188ee4a4e6771ad7473d40db33cf40e378R37 Works with Turbopack but fails in webpack indeed.